identities/models.py

```diff
@@ -73,6 +73,26 @@ def serialize_hook(self, hook):
     def __str__(self):
         return str(self.id)
 
+    def save(self, *args, **kwargs):
+        if self.id and not self._state.adding:
+            old_id = Identity.objects.get(pk=self.id)
```

Comment:
Save will only commit the changes to the db when super.save is called at the bottom of the function. self is the object with uncommitted changes and objects.get will give me the current db version.